love/src/components/MainTel/MTDome/MTDomeShutter.jsx

```diff
@@ -55,13 +55,16 @@ export default class MTDomeShutter extends Component {
   render() {
     const width = this.props.width;
     const height = this.props.height;
+    const positionActualDomeAz = this.props.positionActualDomeAz;
+    const positionCommandedDomeAz = this.props.positionCommandedDomeAz;
```

Comment:
I suggest to use object destructuring for these variables:
`const { width, height, positionActualDomeAz, positionCommandedDomeAz } = this.props;`

Reply:
Thanks!